src/main/java/io/mosip/mimoto/service/impl/CredentialServiceImpl.java

```diff
@@ -134,6 +142,10 @@ public ByteArrayInputStream generatePdfForVerifiableCredentials(VCCredentialResp
         return renderVCInCredentialTemplate(data);
     }
 
+    public Boolean verifyCredential(String credential) {
+        return credentialsVerifier.verifyCredentials(credential);
```

Comment:
verifyCredentials method throws exception why it is not mentioned in method as throws exception?

Reply:
added